scripts/package.json

```
@@ -6,6 +6,7 @@
   "author": "Offchain Labs, Inc.",
   "license": "Apache-2.0",
   "dependencies": {
+    "@arbitrum/nitro-contracts": "^2.1.0",
```

This version would have the timeboost contracts and you don't need to do the dev-contract hack
```suggestion
    "@arbitrum/nitro-contracts": "^2.1.1",
```

---

Thanks I have removed the contract hack.